install.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <project name="WPILib Installation" default="install">
-  <ant antfile="FRamework/install.xml" target="install" useNativeBasedir="true" />
+
```

Comment:
This is another place we need to examine the buildfile as a whole.  This may be changed before we merge.